apps/directory/src/components/biobankcards-components/ResultHeader.vue

```diff
@@ -9,32 +9,48 @@
       <tbody>
         <tr>
           <td class="text-nowrap">
-            Organisation(s): {{ biobanksStore.biobankCardsBiobankCount }}
-          </td>
-          <td class="text-nowrap">
-            Collection(s): {{ biobanksStore.biobankCardsCollectionCount }}
+            {{ organisationLabel }}:
+            {{ biobanksStore.biobankCardsBiobankCount }}
           </td>
         </tr>
         <tr>
           <td class="text-nowrap">
-            Service(s): {{ biobanksStore.biobankCardsServicesCount }}
+            {{ collectionLabel }}:
+            {{
+              biobanksStore.biobankCardsCollectionCount +
+              biobanksStore.biobankCardsSubcollectionCount
+            }}
           </td>
+        </tr>
+        <tr>
           <td class="text-nowrap">
-            Subcollection(s): {{ biobanksStore.biobankCardsSubcollectionCount }}
+            {{ serviveLabel }}: {{ biobanksStore.biobankCardsServicesCount }}
           </td>
         </tr>
       </tbody>
     </table>
   </small>
 </template>
 
-<script>
+<script setup lang="ts">
+import { computed } from "vue";
 import { useBiobanksStore } from "../../stores/biobanksStore";
 
-export default {
-  setup() {
-    const biobanksStore = useBiobanksStore();
-    return { biobanksStore };
-  },
-};
+const biobanksStore = useBiobanksStore();
+
+const organisationLabel = computed(() => {
+  return biobanksStore.biobankCardsBiobankCount > 1
+    ? "Organisations"
+    : "Organisation";
+});
+
+const collectionLabel = computed(() => {
+  return biobanksStore.biobankCardsCollectionCount > 1
+    ? "Collections"
+    : "Collection";
+});
```

Comment:
I don't think we should add the singular labels. `Collection(s): 0/1` looks better than `Collection: 0/1`. I'd say we could just make it always plural even, since it's a count. Expanding the text would be something like `Number of collections: ` So always plural is the way to go i.m.o.